components/ItemCategoryIcon.js

```diff
@@ -1,25 +1,54 @@
-import { TbSofa } from "react-icons/tb";
-import { MdMonitor } from "react-icons/md";
-import { RiTShirt2Line } from "react-icons/ri";
-import { HiOutlineWrenchScrewdriver } from "react-icons/hi2";
-import { GiMedicines } from "react-icons/gi";
-import { BsFillQuestionSquareFill } from "react-icons/bs";
-
 export default function ItemCategoryIcon({ category }) {
   function getIconByCat(category) {
```

Comment:
```suggestion
  function getIconByCategory(category) {
```

i wouldn't unnecessarily shorten names insignificantly, if you pay by reducing readability